docs/install/conda.rst

```diff
@@ -14,9 +14,24 @@ option when installing conda.
 --------------------------
 Binary Package (For Users)
 --------------------------
+
+To list all of the versions of PyNE available on `conda-forge
+<https://conda-forge.github.io/>`_ channel, in your terminal window run::
+
+    conda search -c conda-forge pyne
+
 Binary distributions of the latest release for mac and linux (64-bit) 
```

Comment:
We will not have mac versions available immediately.  This should probably be corrected.